include/cursor.h

```diff
@@ -84,6 +84,15 @@ void cursor_set(struct seat *seat, const char *cursor_name);
 uint32_t cursor_get_resize_edges(struct wlr_cursor *cursor,
 	struct cursor_context *ctx);
 
+/**
+ * cursor_get_from_edge - translate wlroots edge enum to lab_cursor enum
+ * @edge - wlroots edge enum like WLR_EDGE_TOP | WLR_EDGE_RIGHT
+ *
+ * Returns LAB_CURSOR_DEFAULT if edge is neither
+ * one of the 4 corners nor one of the 4 edges.
+ */
+enum lab_cursors cursor_get_from_edge(enum wlr_edges edge);
```

Comment:
Missed that one, will change it to `uint32_t`. I have another branch that changes everything to `enum wlr_edge` but I don't want to mix it in inhere.